images/README.md

```diff
@@ -0,0 +1,119 @@
+# salt-toaster
+
+Build images for salt-toaster
+
+## Setup
+
+The setup is pretty typical for small Python projects. Just clone the
+repository, create a virtual environment, activate it and install the dependencies.
+
+If you choose a different name for your virtual environment, you need to specify
+it later as `VENV` when you use `make`.
+``` sh
+git clone https://gitlab.suse.de/galaxy/toaster-docker-support.git
+cd toaster-docker-support
+python3 -m venv sandbox
+echo "*" > sandbox/.gitignore
+. sandbox/bin/activate
+python -m pip install -r images/requirements.txt
+```
+
+## Description
+
+There are predefined flavors of salt packages plus a `devel` flavor.
+The predefined flavors are packages served from OBS, see the (incomplete) list below:
+
+ - [products](https://build.opensuse.org/package/show/systemsmanagement:saltstack:products/salt)
+ - [products:testing](https://build.opensuse.org/package/show/systemsmanagement:saltstack:products:testing/salt)
+ - [products:next](https://build.opensuse.org/package/show/systemsmanagement:saltstack:products:next/salt)
+ - [products:next:testing](https://build.opensuse.org/package/show/systemsmanagement:saltstack:products:next:testing/salt)
+
+The `devel` flavor means:
+1. archive `SALT_REPO` and push it to container:/salt/src/salt-devel
+2. install Salt using `pip install -e /salt/src/salt-devel`
+
+This allows testing Salt from a local repository.
+
+Example (run in `salt-toaster` folder):
+``` sh
+make docker_shell VERSION=sles12sp2 FLAVOR=devel SALT_REPO=/home/store/repositories/salt NOPULL=true
+```
+In the future, `VERSION` will also be renamed to `DISTRO`. This is not the case
+yet, hence the discrepancy between this project and `salt-toaster`.
```

Comment:
"this project" will be "salt-toaster". A quick follow-up PR could change `VERSION` to `DISTRO` (well, plus an update to the jenkins definitions)

Reply:
Okay this I will include in this PR because it affects the CI and I have changed this the other way around currently.